editions/tw5.com/tiddlers/filters/syntax/Legacy Filter Run Prefix.tid

```diff
@@ -0,0 +1,30 @@
+created: 20230305131705188
+modified: 20230319182200995
+tags: [[Filter Run Prefix]]
+title: Shortcut Filter Run Prefix
+type: text/vnd.tiddlywiki
+
+Parents: <$macrocall $name=".recursiveParentTag" current=<<currentTiddler>> stop="Filters"/>
+
+Shortcut prefixes are commonly used by advanced users because they are fast to type but they are harder to read by less experienced users. That's why [[named prefixes|Named Filter Run Prefix]] have been created, which are more verbose. Shortcut and named filter run prefixes are interchangeable as shown in the table below.
+
+<$railroad text="""
+\start none
+(-|:"+"|"-"|"~"|"=")
+[[run|"Filter Run"]]
+"""/>
```

Comment:
We do not have a possibility to draw extra borders or something similar to make it visible that there is a "higher" level .. I may add a `\start double and \end double` to the Filter Expression. Which will then be the only box with this start and end. 

The important info is the open end anyway, since it tells the users that they should "look up" in the hierarchy.